src/NetworkViewUi.lua

```diff
@@ -163,45 +181,125 @@ function M.update_items(player_index)
   })
   item_flow.style.size = { width = M.WIDTH - 30, height = M.HEIGHT - 82 }
 
-  local rows = M.get_rows_of_items(net_view.view_type)
+  local h_stack_def = {
+    type = "flow",
+    direction = "horizontal",
+  }
+
+  -- pad the item row so there is a slot to dump items from the cursor
+  local did_pad = false
+  local function pad_item_row(item_h_stack, count)
```

Comment:
What do you think about adding the pad item sprite as the first cell in the grid rather than the last row?

Reply:
I'd like that.  I skipped that because I'd have to add it to the "get list" function. Might not be a bad idea.
I also tried clicking anywhere in the background, but I couldn't get that to work.